src/Castle.Windsor.Tests/ReferenceTracker.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright 2018 Castle Project - http://www.castleproject.org/
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace CastleTests
+{
+	using System;
+
+	using NUnit.Framework;
+
+	public static class ReferenceTracker
+	{
+		/// <summary>
+		/// <para>
+		/// Use a lambda method to evaluate an expression returning an object instance to track.
+		/// </para>
+		/// <para>
+		/// (If the expression is evaluated directly inside the test method, the runtime will keep it alive
+		/// until the test method returns if the runtime is in debug mode. ReSharper’s test runner runs in
+		/// debug mode when the solution is configuration is Debug.)
```

Comment:
`when the solution is configuration is Debug` (2x `is`) &rarr; `when the solution's active configuration is Debug` or perhaps even just `when the Debug configuration is active`